libs/langchain-openai/src/tests/azure/chat_models.int.test.ts

```diff
@@ -0,0 +1,829 @@
+import { test, jest, expect } from "@jest/globals";
+import {
+  BaseMessage,
+  ChatMessage,
+  HumanMessage,
+  SystemMessage,
+} from "@langchain/core/messages";
+import { ChatGeneration, LLMResult } from "@langchain/core/outputs";
+import { ChatPromptValue } from "@langchain/core/prompt_values";
+import {
+  PromptTemplate,
+  ChatPromptTemplate,
+  HumanMessagePromptTemplate,
+  SystemMessagePromptTemplate,
+} from "@langchain/core/prompts";
+import { CallbackManager } from "@langchain/core/callbacks/manager";
+import { NewTokenIndices } from "@langchain/core/callbacks/base";
+import { InMemoryCache } from "@langchain/core/caches";
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+import {
+  ClientSecretCredential,
+  getBearerTokenProvider,
+} from "@azure/identity";
+import { AzureChatOpenAI } from "../../azure/chat_models.js";
+
+test("Test ChatOpenAI", async () => {
```

Comment:
All the test names need updating to be representative of what is being tested.

Reply:
Thanks for the feedback. I have updated all the test names representing what is being tested.